jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1406,17 +1406,17 @@ properties:
             fits_keyword: BARTDELT
             blend_table: True
           barycentric_expstart:
-            title: Barycentric exposure start time
+            title: Barycentric exposure start time in MJD
```

Comment:
The latest on-line version of the KWD has the comment field of all these keywords as "[d] Barycentric exposure start time in MJD", so we should make ours match.

Reply:
No, the "[d]" indicates units of days.